app/models/cms/email_template.rb

```diff
@@ -50,7 +50,25 @@ def self.reset_templates_path!
   EMAIL_FORMAT = /(?:(#{EMAIL_ADDRESS_FORMAT})|#{EMAIL_WITH_NAME_FORMAT})\s*/i
   EMAILS_FORMAT = /\A#{EMAIL_FORMAT}(,\s*#{EMAIL_FORMAT})*\Z/i
 
-  serialize :options, Hash
+  class HashOrParameters < ::ActiveRecord::Coders::YAMLColumn
+    def load(string)
+      obj = super(string)
+      obj.respond_to?(:to_unsafe_h) ? obj.to_unsafe_h : obj.to_h if obj
```

Comment:
This is to load serialized `ActionController::Parameters` as `ActiveSupport::HashWithIndiffferentAccess`

Reply:
Maybe it should be `.with_indifferent_access` instead of `.to_h`

@3scale/system-ruby thoughts? 🤔

Reply:
I agree with the `with_indifferent_access`